packages/notebook-extension/package.json

```diff
@@ -24,10 +24,11 @@
     "url": "https://github.com/jupyterlab/jupyterlab.git"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -b src",
```

Comment:
There are several places we have `tsc -b src` instead of `tsc -p src`. Did you mean to do `-p` instead of `-b`?

Reply:
I left the handling of tests for the jest refactor.